Gemfile.lock

```diff
@@ -384,28 +376,26 @@ GEM
       json (>= 1.8, < 3)
       simplecov-html (~> 0.10.0)
     simplecov-html (0.10.2)
-    slim (3.0.8)
+    slim (3.0.9)
```

Comment:
Same here

Reply:
See my comment on this package on the document I gave you on this topic. Slim 3.0.9 fixes an XSS security issue.